editor/src/clj/editor/resource.clj

```diff
@@ -433,20 +433,62 @@
         (io/copy in f))
       (.getAbsolutePath f))))
 
+(def ^:private ext->style-class-suffix
+  {;; Script files
+   "fp" "script"
+   "gui_script" "script"
+   "lua" "script"
+   "render_script" "script"
+   "script" "script"
+   "vp" "script"
+   "glsl" "script"
+
+   ;; Design files
+   "atlas" "design"
+   "collection" "design"
+   "collisionobject" "design"
+   "cubemap" "design"
+   "dae" "design"
+   "font" "design"
+   "go" "design"
+   "gui" "design"
+   "label" "design"
+   "model" "design"
+   "particlefx" "design"
+   "spinemodel" "design"
+   "spinescene" "design"
+   "sprite" "design"
+   "tilemap" "design"
+   "tilesource" "design"
+
+   ;; Property container files
+   "animationset" "property"
+   "camera" "property"
+   "collectionfactory" "property"
+   "collectionproxy" "property"
+   "display_profiles" "property"
+   "factory" "property"
+   "gamepads" "property"
+   "input_binding" "property"
+   "material" "property"
+   "project" "property"
+   "render" "property"
+   "sound" "property"
+   "texture_profiles" "property"})
+
```

Comment:
Wouldn't it be nicer to specify this as
```
{"design" ["atlas"
           "collection"
           "collisionobject"
           ...]
 "property" ["animationset"
             "camera"
             "collectionfactory"
             ...]
 ...}
```
and programmatically map that into this structure?

We should also consider applying the `"resource-ext-"` prefix to the values here to avoid creating garbage strings for each resource later.